docs/explainers/cli-vs-api.md

```diff
@@ -0,0 +1,41 @@
+# Cloud Foundry CLI vs API
+
+## What are the Cloud Foundry CLI and API?
+
+Cloud Foundry provides users several ways to interact with resources outside of applications such as this one: the command line (CLI) and API. The CLI's behavior may be different from similar functionality in the API, and even different versions of the CLI and API may have variations in behavior.
+
+### The versions you may come across
+
+- API
+  - /v2 -- deprecated, still in use by older software such as Stratos
+  - /v3 -- same database as /v2 but very different patterns of access
+- CLI
+  - CLI 7 -- not deprecated but lacks some features of CLI 8
+  - CLI 8 -- introduces async operations, updates to commands
```

Comment:
[question] Do the CLI versions map at all to API versions? (For example, will CLI 7 always run API v2 commands?)

Reply:
It doesn't seem like it, at least not from what I can tell.  I'm still on CLI 7 probably because it's whatever I installed when I first started working here, and more often than not it uses v3 behind the scenes.  I haven't found any documentation that indicates if there is a relationships between the two CLI versions and the API versions, unfortunately.

Reply:
If that's the case, then I'd note here that we haven't found a relationship, and that CLI versions may use both API versions behind the scenes.

Reply:
The way it was written here just made it easy to assume that CLI V7 = API V2, etc